compiler/quilt/test/test_install.py

```diff
@@ -76,8 +78,7 @@ def test_install_latest(self):
         command.install('foo/bar')
         teststore = PackageStore(self._store_dir)
 
-        with open(os.path.join(teststore.package_path(None, 'foo', 'bar'),
-                               Package.CONTENTS_DIR,
+        with open(os.path.join(teststore.package_path(None, 'foo', 'bar'), Package.CONTENTS_DIR,
```

Comment:
Not related to this PR, but should we have a helper for the contents directory? Or maybe, package_path should return the path to the contents directory?

Reply:
Yes, there is a helpers for file validation. I will create a separate PR to clean these tests a little along with new test for installing team based dependencies from .yml